applications/matter_bridge/doc/adding_bridged_matter_device.rst

```diff
@@ -0,0 +1,402 @@
+            }
+
+            void SimulatedPressureSensorDataProvider::NotifyAttributeChange(intptr_t context)
+            {
+               SimulatedPressureSensorDataProvider *provider = reinterpret_cast<SimulatedPressureSensorDataProvider *>(context);
+               provider->NotifyUpdateState(Clusters::PressureMeasurement::Id,
+                           Clusters::PressureMeasurement::Attributes::MeasuredValue::Id,
+                           &provider->mPressure, sizeof(provider->mPressure));
+            }
+
+   #. Implement the body of the :c:func:`NotifyUpdateState` method that shall be called after every data change related to the Pressure Sensor device.
+      It is used to inform the ``Bridge Manager`` and Matter Data Model that an attribute value should be updated.
+      To make the method invoke the appropriate callback, edit the :file:`simulated_pressure_sensor_data_provider.cpp` file as follows:
+
+      .. code-block:: C++
+
+         void SimulatedPressureSensorDataProvider::NotifyUpdateState(chip::ClusterId clusterId, chip::AttributeId attributeId,
+                                    void *data, size_t dataSize)
+         {
+            if (mUpdateAttributeCallback) {
+               mUpdateAttributeCallback(*this, Clusters::PressureMeasurement::Id,
+                        Clusters::PressureMeasurement::Attributes::MeasuredValue::Id, data,
+                        dataSize);
+            }
+         }
+
+   #. Implement the body of the :c:func:`UpdateState` method.
+      This will be called by the ``Bridge Manager`` to inform that data in Matter Data Model was changed and request propagating this information to the end device.
+      In this case, there is no attribute supporting write operations and sending data to end device is not required, so edit the :file:`simulated_pressure_sensor_data_provider.cpp` file as follows:
+
+      .. code-block:: C++
+
+         CHIP_ERROR SimulatedPressureSensorDataProvider::UpdateState(chip::ClusterId clusterId, chip::AttributeId attributeId,
+                                    uint8_t *buffer)
+         {
+            return CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE;
+         }
+
+3. Add the ``PressureSensorDevice`` and ``SimulatedPressureSensorDataProvider`` implementations created in previous steps to the compilation process.
+   To do that, edit the :file:`CMakeLists.txt` file, as follows:
+
+   .. code-block:: cmake
+
+      target_sources(app PRIVATE
+        src/bridged_device_types/pressure_sensor.cpp
+        src/simulated_providers/simulated_pressure_sensor_data_provider.cpp
+      )
+
+4. Provide allocators for ``PressureSensorDevice`` and ``SimulatedPressureSensorDataProvider``  object creation.
+   The Matter Bridge application uses a :c:struct:`SimulatedBridgedDeviceFactory` factory module that creates a pair of ``Matter Bridged Device`` and ``Bridged Device Data Provider`` objects matching a specific Matter device type ID.
```

Comment:
```suggestion
   The Matter Bridge application uses a :c:struct:`SimulatedBridgedDeviceFactory` factory module that creates paired ``Matter Bridged Device`` and ``Bridged Device Data Provider`` objects matching a specific Matter device type ID.

```
Looking at this more, just so it doesn't sound there's going to two of each object.

Empty line to separate the next line to its own.